code/src/components/TodoList.css

```diff
@@ -0,0 +1,112 @@
+.list-of-items-container {
+    padding: 0px 10px;
+    overflow: scroll;
+    max-height: 370px;
+  }
+  
+  @media only screen and (min-height: 569px) {
+    .list-of-items-container {
+      max-height: 465px;
+    }
+  }
+  
+  @media only screen and (min-height: 668px) {
+    .list-of-items-container {
+      max-height: 530px;
+    }
+  }
+  
+  @media only screen and (min-height: 738px) {
+    .list-of-items-container {
+      max-height: 500px;
```

Comment:
hmmm I made more todos just to test the scroll. In Ipad view, you can see the list go outside of the container. I tried adjusting it to just 404px to remove the spillage.

Reply:
I suggest to play more with media queries bec it gets really messy really quickly when you get to different view sizes